### DISCUSSION TITLE
Action List keeps flickering

### DISCUSSION THREAD

**Discussion by Opeyimika-sudo** [illacloud/illa-builder#3758] [Q&A]
Hi guys, when I click on the "+ New" button on the Action List, the resulting list keeps flickering and no resource is clickable from "create a resource" to "transformer". For the first action, I can at least select from the display of "Rest API" and the see more buttons available. After the first action creation, that method is not possible.

This means that for example, if I'm interacting with a Supabase DB for the SELECT action, if I want to add that of the INSERT action, I duplicate the SELECT action and fix it, otherwise I would not be able to pick a new action.

Here's a pictorial evidence. 
https://github.com/illacloud/illa-builder/assets/54380080/a5dd92bf-368d-4661-9a7f-a0ebf166d514
Please I'd appreciate your urgent help as I am working on a client's project.

**Comment by karminski** [comment]
@Opeyimika-sudo Hello, I watched the video and noticed that there may be an issue with the screen resolution. Could you kindly adjust it to a minimum of 1920x1080?

**Comment by Opeyimika-sudo** [comment]
On talking to @karminski in the Discord channel, I found out that my screen resolution was already on 1920 * 1080 but my scale was 150%. When I changed my scale to 100%, the flickering stopped.

Thank you, @karminski.
